source/guide_tarsnap.rst

```diff
@@ -0,0 +1,232 @@
+
+ [isabell@stardust ~]$ tarsnap-keygen \
+							--keyfile /home/$USER/tarsnap/tarsnap.key \
+							--user your_registered_email_from_the_account_registration \
+							--machine some_nifty_name
+							--passphrased
+ [isabell@stardust ~]$
+
+As this key has all rights to manage your backups, you have to create a key with write-only access to do backups automatically per cronjob.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tarsnap-keymgmt \
+							--outkeyfile /home/$USER/tarsnap/tarsnapwrite.key \
+							-w /home/$USER/tarsnap/tarsnap.key``
+ [isabell@stardust ~]$
+
+.. warning:: Please copy your keyfiles to a safe place! Without the keys you cannot access your backups anymore.
+
+
+Set up the config file
+----------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ cp /home/$USER/etc/tarsnap.conf.sample /home/$USER/etc/tarsnap.conf
+ [isabell@stardust ~]$
+
+Now edit ``/home/$USER/etc/tarsnap.conf`` with the editor of your choice. The config file is already commented by the author.
+Make sure to use your write-only keyfile under keyfile in the config file.
+
+
+Create a backup script
+----------------------
+
+Create a script ``/home/$USER/tarsnap-backup.sh`` using the editor of your choice with the following content:
+
+``!/bin/sh
+/home/$USER/bin/tarsnap -c \
+	-f "BACKUP-$(date +%d-%m-%Y_%H-%M-%S)" \
+	/var/www/virtual/$USER``
+
+Now make it executable.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ chmod u+x /home/$USER/tarsnap-backup.sh
+ [isabell@stardust ~]$
+
+
+Setup automatic backups per cronjob
+-----------------------------------
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ crontab -e
+ [isabell@stardust ~]$
+
+Enter ``0 02 * * * /home/$USER/tarsnap-backup.sh`` to let the backup run every night at 2 am.
+
+.. hint:: For help setting up cronjobs go to https://crontab.guru/
+
+
+Test your backup
+----------------
+
+Start a backup using
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ /home/$USER/tarsnap-backup.sh
+ [isabell@stardust ~]$
+
+To show all your existing backups use
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tarsnap --list-archives --keyfile /home/$USER/tarsnap/tarsnap.key | sort
+ [isabell@stardust ~]$
+
+You should see one backup at the moment.
+
+To restore this backup, create another directory as a testing destination using
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ mkdir /home/$USER/restoretest
+ [isabell@stardust ~]$
+ 
+Then use
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ tarsnap -x -v -f BACKUP --keyfile /home/$USER/tarsnap/tarsnap.key -C /home/$USER/restoretest
+ [isabell@stardust ~]$
+
+to restore your backed up files to the testing directory.
+
+.. note:: BACKUP has to be replaced by the name listed by --list-archives in the step above.
```

Comment:
The bare `--` gets mangled by sphinx, see [rendered guide](https://deploy-preview-968--uberlab.netlify.app/guide_tarsnap.html#test-your-backup).
```suggestion
.. note:: BACKUP has to be replaced by the name listed by ``--list-archives`` in the step above.
```